ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-master.yaml

```diff
@@ -13,6 +13,31 @@ images:
 - dockerfile_path: containers/ci/Dockerfile
   from: src
   to: openstack-operator-ci-tooling
+- dockerfile_literal: |
```

Comment:
We need this base image to build and deploy operators, so we need to manage to have that in a common repo (and not duplicating on all repos). Still trying to figure it out the best way of doing that.